### DISCUSSION TITLE
Remix: Deployment failure - Could not resolve "react"

### DISCUSSION THREAD

**Discussion by andrephilipsson** [vercel/community#1723] [Help] [answered]
### Summary

Getting build errors trying to deploy a brand new Remix app. I have attached a reproduction created using:

```sh
pnpm dlx create-remix@latest --template vercel --typescript
```

### Additional information

Errors from the build logs:
```
build: ../../var/task/node_modules/@remix-run/dev/dist/config/defaults/entry.client.react.tsx:1:29: ERROR: Could not resolve "@remix-run/react"
build: ../../var/task/node_modules/@remix-run/dev/dist/config/defaults/entry.client.react.tsx:2:44: ERROR: Could not resolve "react"
build: ../../var/task/node_modules/@remix-run/dev/dist/config/defaults/entry.client.react.tsx:3:28: ERROR: Could not resolve "react-dom/client"
build: ../../var/task/node_modules/@remix-run/dev/dist/config/defaults/entry.client.react.tsx:10:8: ERROR: Could not resolve "react/jsx-runtime"
```

### Example

https://github.com/badapeab/remix-vercel-deploy-reproduction

**Comment by amyegan** [comment, marked as answer]
Thanks for bringing this up, André. The errors you see when running from that template are being addressed in https://github.com/vercel/vercel/pull/9620

As an alternative to that template, you can use [Vercel CLI](https://vercel.com/docs/cli) to create a new Remix starter site by running `vercel init remix`
